src/apollo/client.ts

```diff
@@ -39,6 +39,24 @@ export const hederaMainnetSubgraphGql = new GraphQLClient(
   },
 );
 
+export const avalancheBlockSubgraph = new GraphQLClient(
+  'https://api.thegraph.com/subgraphs/name/dasconnor/avalanche-blocks',
+  {
+    headers: {},
+  },
+);
+
+export const hederaTestnetBlockSubgraph = new GraphQLClient(
+  'https://hedera-test.pangolin.network/subgraphs/name/blocks',
+  {
+    headers: {},
+  },
+);
+
+export const hederaBlockSubgraph = new GraphQLClient('https://hedera-graph.pangolin.network/subgraphs/name/blocks', {
+  headers: {},
+});
```

Comment:
we dont need separate block clients, block query can be done on main subgraph directly